src/index.ts

```diff
@@ -1,2 +1,4 @@
 export { default } from "./useTempData";
 export { default as withTempData } from "./withTempData";
+export { default as reducer } from "./reducer";
```

Comment:
@PejmanNik  What do you think if we export the reducer as tempDataReducer name? This way the user can import the reducer directly without changing the name in his/her reducers file.

Reply:
Good point, I totally agree.